cloud-controller-manager/do/resources.go

```diff
@@ -34,62 +35,211 @@ import (
 )
 
 const (
-	controllerSyncPeriod = 1 * time.Minute
-	requestTimeout       = 10 * time.Second
+	controllerSyncTagsPeriod      = 1 * time.Minute
+	controllerSyncResourcesPeriod = 1 * time.Minute
+	requestTimeout                = 1 * time.Minute
+	syncResourcesTimeout          = 3 * time.Minute
 )
 
 type tagMissingError struct {
 	error
 }
 
+type resources struct {
+	dropletIDMap      map[int]*godo.Droplet
```

Comment:
Personally, I like to name maps by their lookup key, so in this case `dropletsByID`. This avoids encoding the type into the name (which could change) and makes it clear what is being mapped from and where to.